login_page.php

```diff
@@ -0,0 +1,28 @@
+<?php
+session_start();
+?>
+
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <title>Log in to Kate's Portfolio CMS</title>
+    <link rel='stylesheet' type='text/css' href='css/normalize.css'/>
+    <link href="https://fonts.googleapis.com/css?family=Quicksand:300,500" rel="stylesheet">
+    <link rel='stylesheet' type='text/css' href='css/cms_styles.css'/>
+</head>
+<body>
+<h1>Log in to Manage Content for Kate's Portfolio Page</h1>
+    <form method="POST" action="cms_index.php">
+        <label for="username"><h4>Username:</h4></label>
+        <input type = "text" class="typeText" name="username">
```

Comment:
remove spaces between type and = and text

Reply:
done